build/musicbrainz/scripts/fetch-dump.sh

```diff
@@ -98,6 +98,59 @@ then
 	exit 64 # EX_USAGE
 fi
 
+# Show information about signing up for data use
+
+if [[ $TARGET != sample &&
+	! -a "$DB_DUMP_DIR/.for-commercial-use" &&
+	! -a  "$DB_DUMP_DIR/.for-non-commercial-use"
+]]
+then
+	prompt=$(cat <<-EOQ
+		The data you are about to download is provided by the MetaBrainz Foundation.
+		Are you planning to use this data for commercial or business purposes?
+		(y/n)
+	EOQ
+	)
+	read -e -p "$prompt " -r
+	while [[ ! ${REPLY:0:1} =~ [YNyn] ]]
+	do
+		read -e -p "Invalid reply. Yes or no? " -r
+	done
+	echo
+	if [[ ${REPLY:0:1} =~ [Yy] ]]
+	then
+		prompt=$(cat <<-EOQ
+			The MetaBrainz Foundation is supported by commercial users of our data and
```

Comment:
I think that the duplication of the text is not great, since that is bound to get out of sync. Would it be possible to have the text defined in a single file, that then gets used by these two files?

Reply:
It is a more general known issue that many things are duplicated between build contexts for production and development versions of MusicBrainz server. This script is the same for both but others have small differences. There is no way to use symbolic links here and having a common build context would be more overhead.